wfe2/wfe.go

```diff
@@ -843,6 +843,14 @@ func (wfe *WebFrontEndImpl) prepChallengeForDisplay(request *http.Request, authz
 	if challenge.Error != nil && !strings.HasPrefix(string(challenge.Error.Type), probs.V1ErrorNS) {
 		challenge.Error.Type = probs.V2ErrorNS + challenge.Error.Type
 	}
+
+	// DNS-01-Wildcard challenge types are used internally to hint that an
```

Comment:
This is more than a hint; we need to store the data in order to correctly process CAA, for instance.